doc/api/async_hooks.md

```diff
@@ -6,6 +6,14 @@
 
 <!-- source_link=lib/async_hooks.js -->
 
+This module will most likely never reach stable state. Please migrate to
+other APIs.
```

Comment:
I think this PR and #45335 are actually quite complementary. I think the statement about the future status of `async_hooks` is best put in the block that currently says “Status: Experimental” (the part changed in the other PR). Maybe we could change this sentence here to be something more introductory about the migration path, leaving the statement for the other PR; and we can continue to revise that other PR into whatever “future status” statement we think is best?

```suggestion
Some stable APIs that cover many of the `async_hooks` use cases are:
```

Reply:
Moved a part and changed wording a bit.